services/common/boltzmann/README.md

```diff
@@ -120,8 +120,8 @@ async function greeting(context, params) {
   return boltzmann.response.text(`greetings, ${params.human}`);
 }
 
-const router = boltzmann.fork.router()(
-    fork.get('/greet/:human', greeting)
+const router = boltzmann.fork.router() (
+  fork.get('/greet/:human', greeting)
```

Comment:
What do you think of adding [`ping`](https://github.com/entropic-dev/entropic/blob/0be1a86ca55adcd76b11bd4750513661a3ec6d2b/services/web/handlers/index.js#L11) here too?